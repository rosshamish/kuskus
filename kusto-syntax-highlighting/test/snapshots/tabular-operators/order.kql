// https://learn.microsoft.com/en-us/azure/data-explorer/kusto/query/orderoperator

T | order by country asc, price desc
