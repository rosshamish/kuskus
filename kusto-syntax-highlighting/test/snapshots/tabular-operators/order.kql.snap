>// https://learn.microsoft.com/en-us/azure/data-explorer/kusto/query/orderoperator
#^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.kusto comment.line.kusto
>
>T | order by country asc, price desc
#^^^^ source.kusto
#    ^^^^^ source.kusto meta.query.order.kusto keyword.other.query.kusto
#         ^ source.kusto meta.query.order.kusto
#          ^^ source.kusto meta.query.order.kusto keyword.other.operator.kusto
#            ^ source.kusto meta.query.order.kusto
#             ^^^^^^^ source.kusto meta.query.order.kusto variable.other.column.kusto
#                    ^ source.kusto
#                     ^^^ source.kusto keyword.other.kusto
#                        ^^^^^^^^ source.kusto
#                                ^^^^ source.kusto keyword.other.kusto
>